### PR TITLE
ROBOT: API doc updates - with robot examples

### PR DESCRIPTION
**Pull request by melinda-nagy**
@alttester/developers please review when possible.
Thanks!

### CODE REVIEW

File: Docs/source/pages/commands.md [hunk 1]
```diff
@@ -1353,6 +1565,24 @@ Simulates a swipe action between two points.
                 By.NAME, "Drop").get_component_property("UnityEngine.UI.Image", "sprite", "UnityEngine.UI")
             assert image_source != image_source_drop_zone
 
+    .. code-tab:: robot
+
+        Test Resize Panel
+            ${alt_object}=    Find Object    NAME    Resize Zone
+            ${alt_object_x}=    Get Object X    ${alt_object}
```
Comment (SandyEma9) [inline Docs/source/pages/commands.md:1572]: I would use the same example as other languages.

File: Docs/source/pages/commands.md [hunk 2]
```diff
@@ -2355,6 +2799,17 @@ None
         def test_get_current_scene(self):
             self.altDriver.load_scene("Scene 1 AltDriverTestScene")
             self.assertEqual("Scene 1 AltDriverTestScene",self.altDriver.get_current_scene())
+
+    .. code-tab:: robot
+
+        Test Load And Wait For Scene
+            Load Scene    ${scene1}
+            Wait For Current Scene To Be    ${scene1}    timeout=1
+            Load Scene    ${scene2}
```
Comment (SandyEma9) [inline Docs/source/pages/commands.md:2808]: Do we need to load 2 scenes or is enough just one?

File: Docs/source/pages/commands.md [hunk 3]
```diff
@@ -2401,6 +2856,16 @@ Loads a scene.
             self.altDriver.load_scene("Scene 1 AltDriverTestScene",True)
             self.assertEqual("Scene 1 AltDriverTestScene",self.altDriver.get_current_scene())
 
+    .. code-tab:: robot
+
+        Test Load And Wait For Scene
+            Load Scene    ${scene1}
+            Wait For Current Scene To Be    ${scene1}    timeout=1
+            Load Scene    ${scene2}
```
Comment (SandyEma9) [inline Docs/source/pages/commands.md:2864]: Do we need to load 2 scenes or is enough just one?

**Review by rucindrea — APPROVED** [review]

File: Docs/source/pages/commands.md [hunk 4]
```diff
@@ -764,6 +870,21 @@ Simulates a key down.
             lastKeyUp = self.altDriver.find_object(By.NAME, 'LastKeyUpValue')
             self.assertEqual("A", lastKeyUp.get_text())
 
+    .. code-tab:: robot
+
+        Test Key Down And Key Up
+            Key Down    A
```
[the remark this answers is not in the document]
Comment (melinda-nagy) [reply Docs/source/pages/commands.md:871]: Sure, we can do that 👍 . In the robot tests the "Load Scene" part was outside of the tests, in the Setup.

File: Docs/source/pages/commands.md (hunk 1 above)
Comment (melinda-nagy) [reply Docs/source/pages/commands.md]: Used corresponded test example from robot library.

**Review by rucindrea — APPROVED** [review]

**Review by SandyEma9 — APPROVED** [review]

**Pull request merged**